liquibase-standard/src/main/java/liquibase/analytics/CsvAnalyticsListener.java

```diff
@@ -0,0 +1,32 @@
+package liquibase.analytics;
+
+import liquibase.Scope;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class CsvAnalyticsListener implements UsageAnalyticsListener {
```

Comment:
This class should be removed, along with its enum once the Segment listener is added.